cluster/contrail-collector.manifest

```diff
@@ -0,0 +1,38 @@
+{
+    "apiVersion": "v1",
+    "kind": "Pod",
+    "metadata": {"name":"contrail-coll"},
```

Comment:
Apparently, there is size limit for the name :(



On Wed, Sep 2, 2015 at 4:24 PM, Pedro Marques <notifications@github.com>
wrote:

> In cluster/contrail-collector.manifest
> <https://github.com/Juniper/contrail-kubernetes/pull/40#discussion_r38597755>
> :
>
> > @@ -0,0 +1,38 @@
> > +{
> > +    "apiVersion": "v1",
> > +    "kind": "Pod",
> > +    "metadata": {"name":"contrail-coll"},
>
> contrail-collector
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/40/files#r38597755>.
>